pubsub/callback/callback.go

```diff
@@ -0,0 +1,41 @@
+package callback
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/micromdm/micromdm/pubsub"
+)
+
+type CallbackPublisher struct {
+	url  string
+	next pubsub.Publisher
+}
+
+func (p *CallbackPublisher) Publish(ctx context.Context, topic string, msg []byte) error {
+
+	// send to callback url
+	var buf bytes.Buffer
+	var event = struct {
+		Topic   string `json:"topic"`
+		Message []byte `json:"message"`
+	}{
+		Topic:   topic,
+		Message: msg,
```

Comment:
The one thing I still don't agree with is that JSON makes a better option here over plist. 

The reason being is that `micromdm/mdm` still has a long way to go before we have accurate structs for ALL responses. 
if we publish the raw plist as it comes to `/connect` there would be a greater value to user right now, even if their callback handler has to decode a plist.